site-src/implementations.md

```diff
@@ -226,11 +226,14 @@ FSM support of Gateway API is built on top [Flomesh Gateway API](fgw) and it cur
 [flomesh]:https://flomesh.io
 [fgw]:https://github.com/flomesh-io/fgw
 
-### Gloo Edge
+### Gloo Gateway
 
-Gloo Edge 2.0 is an Istio-native, fully-featured Envoy based API gateway that brings [Gloo Edge][glooedge] functionality to Istio. The [Solo.io][solo] team is actively working towards an implementation of the Gateway API.
+[![Conformance](https://img.shields.io/badge/Gateway%20API%20Conformance%20v1.0.0-GlooGateway-green)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.0.0/gloo-gateway.yaml)
 
-[glooedge]:https://docs.solo.io/gloo-edge/
+[Gloo Gateway][gloogateway] by [Solo.io][solo] is a feature-rich, Kubernetes-native ingress controller and next-generation API gateway.
+Gloo Gateway 2.0 brings the full power and community support of the Gateway API to it's existing control-plane implementation.
```

Comment:
Very tiny nit:

```suggestion
Gloo Gateway 2.0 brings the full power and community support of Gateway API to it's existing control-plane implementation.
```

x-ref: https://github.com/kubernetes-sigs/gateway-api/issues/2553

Reply:
If only programing also improved my grammar 😆, thank you @robscott